README.md

```diff
@@ -93,7 +93,12 @@ Each country has a GHG mass flow that depends on neighboring countries. In order
   - Gas: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show)
   - Hydro: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show)
   - Wind: [ieawind.org](http://www.ieawind.org/countries/norway.html)  
-- Poland: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show)
+- Poland: 
+  - Biomass : [Ure.gov.pl](https://www.ure.gov.pl/pl/rynki-energii/energia-elektryczna/odnawialne-zrodla-ener/potencjal-krajowy-oze/5753,Moc-zainstalowana-MW.html)
+  - Hydro : [Ure.gov.pl](https://www.ure.gov.pl/pl/rynki-energii/energia-elektryczna/odnawialne-zrodla-ener/potencjal-krajowy-oze/5753,Moc-zainstalowana-MW.html)
+  - Solar : [Ure.gov.pl](https://www.ure.gov.pl/pl/rynki-energii/energia-elektryczna/odnawialne-zrodla-ener/potencjal-krajowy-oze/5753,Moc-zainstalowana-MW.html)
+  - Wind : [Ure.gov.pl](https://www.ure.gov.pl/pl/rynki-energii/energia-elektryczna/odnawialne-zrodla-ener/potencjal-krajowy-oze/5753,Moc-zainstalowana-MW.html)
```

Comment:
Weird, for some reason the `-` is not formatted correctly. In the preview it also has bad formatting :(